src/Sulu/Bundle/PageBundle/Markup/Link/ExternalLinkProvider.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\PageBundle\Markup\Link;
```

Comment:
External links should also work when only have the admin bundle. So We should move this to the admin bundle.